sklearn/linear_model/cd_fast.pyx

```diff
@@ -794,5 +817,11 @@ def enet_coordinate_descent_multi_task(floating[::1, :] W, floating l1_reg,
                 if gap < tol:
                     # return if we reached desired tolerance
                     break
+                else:
```

Comment:
after this warning message was printed, the for-loop goes on if ``max_iter`` is not reached, right?
And if ``max_iter`` is reached before the condition in 767 happens then it won't converge but never warn?